client/src/components/Slider/Slider.js

```diff
@@ -0,0 +1,107 @@
+import imageFile1 from "../../Images/Fall-2020-1.png";
+import imageFile2 from "../../Images/Fall-2020-2.png";
+import imageFile3 from "../../Images/Spring-2020-1.jpeg";
+import imageFile4 from "../../Images/Spring-2020-2.jpeg";
+import imageFile5 from "../../Images/Trending-Now-1.jpeg";
+import imageFile6 from "../../Images/Trending-Now-2.jpeg";
+import imageFile7 from "../../Images/Classics-1.jpeg";
+import imageFile8 from "../../Images/Classics-2.jpeg";
+
+import React from "react";
+import ReactDOM from "react-dom";
+import "./Slider.css";
+
+const slides = {
+  properties: [
+    {
+      title: "Fall 2020 Collection",
+      imga: imageFile1,
+      imgb: imageFile2,
+      labela: "LUCA",
+      labelb: "VIDA",
+      index: 0
+    },
+    {
+      title: "Spring 2020 Collection",
+      imga: imageFile3,
+      imgb: imageFile4,
+      labela: "AlWAYS",
+      labelb: "BELIZE",
+      index: 1
+    },
+    {
+      title: "Trending Now",
+      imga: imageFile5,
+      imgb: imageFile6,
+      labela: "ALICE",
+      labelb: "EVELYN",
+      index: 2
+    },
+    {
+      title: "Classics",
+      imga: imageFile7,
+      imgb: imageFile8,
+      labela: "AINSLEY",
+      labelb: "ANNIE",
+      index: 3
+    }
+  ]
+};
+
+const Card = ({ property }) => {
+  const { title, imga, imgb, labela, labelb, index } = property;
+  return (
+    <div id={`slide-${index}`} className="slide">
+      <p className="title">{`${title}`}</p>
+      <div className="slide-container">
+        <div className="image-1">
+          <img src={imga} />
+          <p>{`${labela}`}</p>
+        </div>
+        <div className="image-2">
+          <img src={imgb} />
+          <p>{`${labelb}`}</p>
+        </div>
+      </div>
+    </div>
+  );
+};
```

Comment:
There is an existing Card component that you could use instead of defining your own. Otherwise, you could call this something like `Slide` if it's different from that component

Reply:
There is no Card component for this carousel, but I went ahead and renamed that variable to Slide.